security/ldap-config-handler/src/main/java/org/codice/ddf/admin/security/ldap/test/LdapTestingCommons.java

```diff
@@ -194,6 +199,55 @@ private static BindRequest selectBindMethod(String bindMethod, String bindUser,
         return request;
     }
 
+    /**
+     * Checks for existing LDAP configurations with the same hostname and port of the {@code configuration}.
+     * If there is an existing configuration, warnings will be returned. If there are no existing configurations
+     * an empty {@link List} will be returned.
```

Comment:
❗️ Change from `@link` to `@code`. There's no advantage to linking to the Java `List` documentation here when the generated docs for the method will already do so.